internal/cmd/schema-registry/command_config_describe.go

```diff
@@ -75,7 +83,38 @@ func describeSchemaConfig(cmd *cobra.Command, srClient *srsdk.APIClient, ctx con
 		}
 	}
 
+	defaultMetadata, err := json.Marshal(config.DefaultMetadata)
+	if err != nil {
+		return err
+	}
+
+	overrideMetadata, err := json.Marshal(config.OverrideMetadata)
+	if err != nil {
+		return err
+	}
+
+	defaultRuleset, err := json.Marshal(config.DefaultRuleSet)
+	if err != nil {
+		return err
+	}
+
+	overrideRuleset, err := json.Marshal(config.OverrideRuleSet)
+	if err != nil {
+		return err
+	}
+
 	table := output.NewTable(cmd)
-	table.Add(&configOut{CompatibilityLevel: config.CompatibilityLevel})
-	return table.Print()
+	table.Add(&configOut{
+		CompatibilityLevel: config.CompatibilityLevel,
+		CompatibilityGroup: config.CompatibilityGroup,
+		MetadataDefaults:   prettyString(defaultMetadata),
+		MetadataOverrides:  prettyString(overrideMetadata),
+		RulesetDefaults:    prettyString(defaultRuleset),
+		RulesetOverrides:   prettyString(overrideRuleset),
+	})
+	return table.PrintWithAutoWrap(false)
+}
+
+func prettyString(str []byte) string {
```

Comment:
Let's clarify that this is JSON
```suggestion
func printJson(str []byte) string {
```